frontend/src/store/modules/shoots/index.js

```diff
@@ -364,6 +366,35 @@ const actions = {
 
     commit('RESET_NEW_SHOOT_RESOURCE', shootResource)
     return state.newShootResource
+  },
+  setFreezeSorting ({ commit }, value) {
+    commit('SET_FREEZE_SORTING', value)
+
+    if (value) {
+      const freezedShootSkeletons = map(state.filteredShoots, shoot => {
```

Comment:
why "only" skeletons?

Reply:
We only want to copy values required to display the stale items in the table. We could also copy more data or even the whole objects. We should discuss this.